backend/config/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE module PUBLIC
     "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-     "https://www.puppycrawl.com/dtds/configuration_1_3.dtd">
```

Comment:
OK. I'm so confused. I thought I checked this url just a couple of days ago when I changed the url to https.

Reply:
It just started to error on me￼￼ recently. I have no idea what happened there, but the other address works, so... 🤷‍♂️